apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Internal/Logging/Logger.cs

```diff
@@ -26,32 +26,30 @@ namespace Google.Cloud.Spanner.V1.Internal.Logging
     /// </summary>
     public abstract class Logger
```

Comment:
This class is ours as well as `DefaultLogger`, and it doesn't implements any interfaces or other classes that are not "internal".
Why can't we have this class marked as `internal` and the methods as `protected internal` or even just `internal` and then make internals visible across projects.
It's weird with all these comments.

Reply:
Good call - we can make this one internal as nothing else creates a DefaultLogger.

We can't use InternalsVisibleTo between production projects because they can be versioned independently. (We did that before, and it was a mistake.)

Reply:
OK, I see how making InternalsVisibleTo between production projects can be a problem.